### PR TITLE
Install primer.json with black

### PR DESCRIPTION
**Pull request by hroncok**
Fixes https://github.com/psf/black/issues/2153

### CODE REVIEW

**Review by JelleZijlstra — APPROVED** [review]

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 ## Change Log
 
+### Unreleased
+
+#### _Packaging_
+
+- Install `primer.json` with black. (#2153)
```
Comment (cooperlees) [inline CHANGES.md:7]: Bummer ... so close!
```suggestion
- Install `primer.json` with black. (#2154)
```

**Review by cooperlees — CHANGES_REQUESTED** [review]

File: CHANGES.md (hunk 1 above)
Comment (ichard26) [reply CHANGES.md:7]: Might be worth it for mention this is only for black-primer (not general black users)?

Comment (JelleZijlstra) [reply CHANGES.md:7]: It should be the PR number, makes it easier to check automatically that it's correct.

Comment (cooperlees) [reply CHANGES.md]: I updated via web.

Comment (ichard26) [reply CHANGES.md]: Our changelog policy to give the contributor the credit by linking the PR number. I was going to link our changelog policy but I just realized we don't mention that it's the PR number we're looking for...

I should fix that, sorry !

Comment (hroncok) [reply CHANGES.md]: That was non-obvious for me. ack

**Comment by hroncok** [comment]
Amended.

**Comment by cooperlees** [comment]
> Amended.

Thanks - But the fact you had to force push should have set off alarm bells. I had already done the change.

**Comment by hroncok** [comment]
You've only changed the number ;)

**Pull request merged**